mne/viz/tests/test_3d.py

```diff
@@ -615,6 +616,7 @@ def test_plot_source_estimates(renderer_interactive, all_src_types_inv_evoked,
             these_kwargs = kwargs.copy()
             these_kwargs.pop('src')
             meth(**these_kwargs)
+    renderer_interactive.backend._close_all()
```

Comment:
Rather than do this, how about we just have both `renderer_interactive` and `brain_gc` call this after their `yield` steps? Calling it more than once should be fine, and having it in both means we shouldn't need to worry about order.

Reply:
Sure, I'll see if this solves the need to the intermediate calls too